dapp/package.json

```diff
@@ -35,6 +35,7 @@
     "@tanstack/react-query-devtools": "^5.49.2",
     "axios": "^1.6.7",
     "bignumber.js": "^9.1.2",
+    "crypto-js": "^4.2.0",
```

Comment:
Do we need the new lib? I think we can use [ethers](https://docs.ethers.org/v6/api/crypto/#sha256) or [viem](https://viem.sh/docs/utilities/sha256.html).

Reply:
[679db3f](https://github.com/thesis/acre/pull/904/commits/679db3f05a26befb392b3ab2cdfb77eed39155d7)